server/Mistakes.Journal.Api/Api/Mistakes/WebModels/WeatherWebModel.cs

```diff
@@ -0,0 +1,36 @@
+namespace Mistakes.Journal.Api.Api.Mistakes.WebModels
+{
+    public class WeatherWebModel
+    {
+        public WeatherType Weather { get; set; }
+
+        public TemperatureRange Temperature { get; set; }
+
+        public TimeOfDayType TimeOfDay { get; set; }
+
+        public string Place { get; set; }
```

Comment:
What is `Place`? Is it `City`, `Country` or some kind of `Continent`?

Reply:
According to API documentation -> city name